app/views/student_quizzes/take_quiz.html.erb

```diff
@@ -14,6 +14,11 @@
 
   <%= hidden_field_tag :questionnaire_id, "#{@questionnaire.id}" %>
   <%= hidden_field_tag :assignment_id, "#{params[:assignment_id]}" %>
-  <%= render :partial => 'quiz_form', :locals => { :questionnaire_id => @questionnaire.id,:display=>"take" } %>
```

Comment:
quiz_form partial is not used by us any more, We are not sure whether it is used by other views, so we decided to keep it.